test/ContainerOptionFactoryTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LaminasTest\Cli;
+
+use Laminas\Cli\ContainerOptionFactory;
+use PHPUnit\Framework\TestCase;
+
+/** @psalm-suppress PropertyNotSetInConstructor */
+final class ContainerOptionFactoryTest extends TestCase
+{
+    /** @var ContainerOptionFactory */
+    private $factory;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->factory = new ContainerOptionFactory();
+    }
+
+    public function testWillCreateInputOptionWithExpectedName(): void
```

Comment:
These checks didn't exist, so this adds at least some coverage on the `--container` option.